src/interaction-handlers/ticket/categorizeCrash.ts

```diff
@@ -37,11 +37,14 @@ export class ButtonHandler extends InteractionHandler {
         );
       case "launcher":
         return channel.send(`You'll need to send your crash log. Please follow these instructions:
-1. Open your SkyClient folder. See this: https://youtu.be/BHIM2htfMk8
+1. Open your SkyClient folder. If you use the vanilla launcher, see this: https://youtu.be/BHIM2htfMk8
```

Comment:
New users probably won't know what "vanilla launcher" is. "Minecraft Launcher" is probably preferred here.
```suggestion
1. Open your SkyClient folder. If you use Minecraft Launcher, see this: https://youtu.be/BHIM2htfMk8
```